react/queries/getStore.gql

```diff
@@ -5,6 +5,11 @@ query getStore($id: String!) {
       dayOfWeek
       closingTime
     }
+    pickupHolidays {
+      date
+      hourBegin
+      hourEnd
+    }
```

Comment:
vtex.store-graphql already have this fields in the getStore query? if not, this app will break the store details, maybe we have to wait for it before publish this version.